### ISSUE TITLE
UISample

### ISSUE THREAD

**Issue by scharffc** [Amrit-choudhary16/CS6392019#4]
@Amrit-choudhary16 

Hi Amrit

I look at UISample. 

The pictures on the screen are distorted. You should be careful that when you edit the pictures they are not enlarging or flattening the images.

All the pictures on one phone have the same dpi level. It is based on the characteristics of your phone. When you write the different sizes this is not what happened. The dpi level is determined by the Android OS.

**Comment by scharffc** [comment]
![IMG_2942](https://user-images.githubusercontent.com/761057/59784977-ad7bdd80-9291-11e9-938c-6f54293a39a4.jpg)

**Comment by Amrit-choudhary16** [comment]
Yes professor. I realized this mistake of mine that I put in the names of different sizes. But it depends from device to device. My mistake was that I put in the sizes names.

**Comment by Amrit-choudhary16** [comment]
Hello professor, 
Do I need to fix the issue of the distorted image? It was because i edited the pictures and the mistake I made was I displayed the format names which isn’t correct. It depends from device to device. 
Thanks and regards
Amrit choudhary

**Comment by Amrit-choudhary16** [comment]
Hello professor, 
my mistake was in assignment 2b- ImageViewer. The images appeared distorted due to cropping. I have fixed my images and uploaded my file on GitHub again. My images appear clear now. 
Thanks and regards
Amrit choudhary
